package.json

```diff
@@ -33,8 +33,8 @@
     "pkg-clean": "rimraf build out pkg/node_modules pkg/yarn.lock",
     "pkg-all": "source .circleci/local_publish_helpers.sh && generatePkgCli",
     "pkg-all-local": "yarn verdaccio-start && yarn verdaccio-connect && yarn publish-to-verdaccio && yarn pkg-all && yarn verdaccio-disconnect",
-    "publish:dev": "lerna publish --preid=rc.$(git rev-parse --short HEAD) --pre-dist-tag rc --exact --include-merged-tags --no-verify-access --conventional-prerelease --conventional-commits --yes --no-push",
-    "publish:rc": "lerna publish --preid=rc.$(git rev-parse --short HEAD) --pre-dist-tag rc --exact --include-merged-tags --no-verify-access --conventional-prerelease --conventional-commits --yes --no-push",
```

Comment:
@danielleadams did you intend to specify `--include-merged-tags` here? Based on the [docs](https://github.com/lerna/lerna/tree/main/commands/version#--include-merged-tags) it doesn't seem like we need it but perhaps we do?

Reply:
I don't think we need it. When i copied to script over from `master`, it was also using the flag, so I left it https://github.com/aws-amplify/amplify-cli/pull/10703